modules/core/src/test/java/org/apache/synapse/deployers/SynapseArtifactDeploymentStoreTest.java

```diff
@@ -0,0 +1,125 @@
+/*
+ *  Copyright (c) 2017, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+package org.apache.synapse.deployers;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+/**
+ * Test class for SynapseArtifactDeploymentStore
+ */
+public class SynapseArtifactDeploymentStoreTest {
+    /**
+     * Test adding of artifact
+     * @throws Exception
+     */
+    @Test
+    public void testAddArtifact() throws Exception{
```

Comment:
Method signature should format correctly. There should be space before `{`. Please check in the entire PR.

Reply:
Fixed with https://github.com/wso2/wso2-synapse/pull/1100